emacs/merlin-company.el

```diff
@@ -80,6 +80,11 @@
                " (successive calls will expand aliases)"))
       (t default))))
 
+(defun merlin-company--in-string-or-comment-p ()
+  "Return non-nil if START is in a comment or string."
+  (let ((state (syntax-ppss)))
+    (or (nth 3 state) (nth 4 state))))
```

Comment:
That is completely indecipherable to me.
Is there no nicer way to do this?

If not, could we at least get some comments / documentation?